### DISCUSSION TITLE
Install Ventura 13.3.1 Guest on Monterey Host: not supported

### DISCUSSION THREAD

**Discussion by tbussmann** [utmapp/UTM#5286] [macOS]
When I try to create a new virtualized VM and choose the 13.3.1 IPSW `UniversalMac_13.3.1_22E261_Restore.ipsw` downloaded from Apple (sha265sum `6e9d9b30528ec951d8a377173b355932647194c326347efc5e54ade1fe71cbc8`),  I get a message "Your machine does not support running this IPSW.". When using the 13.2.1 IPSW however, the process continues, Ventura installs and runs just fine. I don't understand why the minor Version of the Guest should make a difference in compatibility.

The Host machine is a Monterey 12.6.5 on a 2021 MBP with M1 Max. UTM Version 4.2.5 (81)

**Comment by ANGRYCONE** [comment]
bro's been waiting for 2 years 💔
